src/openforms/appointments/contrib/base.py

```diff
@@ -0,0 +1,163 @@
+import logging
```

Comment:
We usually put the base plugin definition etc. in `openforms.<module>.base` rather than `openforms.<module>.contrib.base`.